src/Components/CartCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useCart } from '../Context/cart-context';
+function CartCard({ CartProduct }) {
+	const { dispatch } = useCart();
+	return (
+		<div className=" flex-row product-card ">
+			<div className="prod-img">
+				<img src={CartProduct.image} alt="" className="responsive-img" />
```

Comment:
always try to mention something in the alt prop for images, best practice.

Reply:
Added the text in alt